python_modules/dagster/dagster/_core/definitions/decorators/sensor_decorator.py

```diff
@@ -120,6 +120,29 @@ def asset_sensor(
             (experimental) A list of jobs to be executed when the sensor fires.
         default_status (DefaultSensorStatus): Whether the sensor starts as running or not. The default
             status can be overridden from Dagit or via the GraphQL API.
+
+
+    Example:
+
+        .. code-block:: python
+
+            from dagster import AssetKey, EventLogEntry, SensorEvaluationContext, asset_sensor
+
+
+            @asset_sensor(asset_key=AssetKey("my_table"), job=my_job)
+            def my_asset_sensor(context: SensorEvaluationContext, asset_event: EventLogEntry):
+                yield RunRequest(
```

Comment:
what do you think about returning instead of yielding?

Hoping to move away from the generator syntax since it misleads folks in terms of the execution pattern.

Reply:
sounds fine to me